resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/v2/ClientV2Suite.scala

```diff
@@ -283,7 +294,7 @@ class ClientV2Suite extends SparkFunSuite with BeforeAndAfter {
     }
   }
 
-  private def getInitContainerSecret(): Secret = {
+  private def getInitContainerSecret: Secret = {
```

Comment:
I think Spark style is to not use `getX` for getters.

http://docs.scala-lang.org/style/naming-conventions.html#methods